addons/mail/static/src/models/emoji_list_view/emoji_list_view.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'EmojiListView',
+    identifyingFields: ['popoverViewOwner'],
+    lifecycleHooks: {
+        _created() {
+            this.onClickEmoji = this.onClickEmoji.bind(this);
+        },
+    },
+    recordMethods: {
+        /**
+         * @param {MouseEvent} ev 
+         */
+        onClickEmoji(ev) {
+            if (this.popoverViewOwner.messageActionListOwnerAsReaction) {
+                this.popoverViewOwner.messageActionListOwnerAsReaction.onClickReaction(ev);
+            }
+            if (this.popoverViewOwner.composerViewOwnerAsEmoji) {
+                this.popoverViewOwner.composerViewOwnerAsEmoji.onClickEmoji(ev);
+            }
```

Comment:
We should probably the API of these methods to directly pass `ev.currentTarget.dataset.unicode`, I don't think other value from `ev` are relevant or desirable to forward.

Reply:
These are "onClick" methods, so maybe we can have some use of marking these events as handled?

Reply:
Well we can rename them to "onEmojiSelected" instead of "on click" if that is what is bothering you with not passing ev.